CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ IF(APPLE)
 	OPTION(WANT_OSC  "Enable OSC support" OFF)
 	OPTION(WANT_JACKSESSION "Enable use of Jack-Session-Handler" OFF)
 ELSE()
-	OPTION(WANT_JACKSESSION "Enable use of Jack-Session-Handler" ON)
+	OPTION(WANT_JACKSESSION "Enable use of Jack-Session-Handler" OFF)
```

Comment:
Should we remove the option completely? This will still be shown in cmake frontends like "ccmake"

Reply:
Sure. Haven't thought about that one.